interface/main/tabs/js/tabs_view_model.js

```diff
@@ -19,6 +19,11 @@ function tabStatus(title,url,name,closable,visible,locked)
     self.title=ko.observable(title);
     self.url=ko.observable(url);
     self.name=ko.observable(name);
+    self.loading_text=ko.observable((xl("Loading") + " " + xl(label) + "..."));
```

Comment:
change xl(label) to label. shouldn't try to xlate variables and label should already be xlated.

Reply:
I will try to do this, but I think it will be really tough when opening a tab that is clicked from the menu.

Reply:
disregard post. it is working without above (because it appears calendar has already translated the label :) )